fmriprep/workflows/bold/base.py

```diff
@@ -294,6 +294,7 @@ def init_bold_wf(
         fieldmap_id=fieldmap_id,
         omp_nthreads=omp_nthreads,
     )
+    # NOTE: Why put this in 'resampling' and not 'full'?
```

Comment:
I believe I was initially going to emit the goodvoxels mask as a "resampling" output, and fsLR resampling is done from anat and not bold, at present. But I think it can be pushed off to "full" now.

Reply:
Do you want me to move it in this PR, or would you prefer a separate one?

Reply:
Sure, you can move it.

Reply:
Done!